Spore ModAPI/Spore/CommonIDs.h

```diff
@@ -93,6 +93,86 @@ enum Difficulty
 	kDifficultyHard = 2,
 };
 
+
```

Comment:
An `enum`  is not a required naming scope. This means just typing `AOA` is using that enum. So this is a lot of names that are suddenly not usable.
Either put them in a namespace, similar to `TypeIDs`, or turn it into a `enum class`.